termennetwerk.ts

```diff
@@ -45,29 +59,93 @@ const graphQlQuery = gql`
   }
 `;
 
+const querySources = gql`
+  query Sources {
+    sources {
+      uri
+      name
+    }
+  }
+`;
+
 const cache: { [query: string]: string } = {};
 
 const client = new GraphQLClient(endpoint);
 
-export async function getUriOfSearchTerm(sources: string[], searchTerm: string): Promise<string> {
-    if (searchTerm === "") return undefined;
-    const cacheKey = searchTerm.toLowerCase();
-    if (cache[cacheKey]) return cache[cacheKey];
-    const response: GraphQlResponse = await client.request(graphQlQuery, { sources, query: searchTerm });
-    const result = response.terms[0].result;
-//    if (!Array.isArray(result.terms) ||   
+function selectTerm(result: Result, searchTerm: string, source: string): Term | undefined {
+
+    if (result.__typename === 'ServerError') { // catch server errors
+      //console.log("Termennetwerk refinement: server error for source",source);
+      return undefined; 
+    }
+
     if (result.__typename === 'TimeoutError') { // catch timeouts errors
-      console.error(`Timeout error from Network-of-Terms API: "${result.message}" for keyword: "${searchTerm}"`);
+      //console.log("Termennetwerk refinement: time out for source", source);
```

Comment:
Why remove the logging?

Reply:
To avoid too much noise, but your right I activated them again.